tests/integration/apigateway/conftest.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+
+from localstack.constants import APPLICATION_JSON
+from localstack.utils.strings import short_uid
+from tests.integration.apigateway_fixtures import (
+    create_rest_api_deployment,
+    create_rest_api_integration,
+    create_rest_api_integration_response,
+    create_rest_api_method_response,
+    create_rest_api_stage,
+    create_rest_resource,
+    create_rest_resource_method,
+)
+from tests.integration.test_apigateway import (
+    APIGATEWAY_ASSUME_ROLE_POLICY,
+    APIGATEWAY_DYNAMODB_POLICY,
+    APIGATEWAY_KINESIS_POLICY,
+)
+
+# default name used for created REST API stages
+DEFAULT_STAGE_NAME = "dev"
+
+
+@pytest.fixture
+def create_rest_api_with_integration(
+    create_rest_apigw,
+    apigateway_client,
+    wait_for_stream_ready,
+    create_iam_role_with_policy,
+):
+    def _factory(
+        integration_uri, req_templates=None, res_templates=None, stage=None, integration_type=None
```

Comment:
if stage is the default I'd consider add it here as default value for the `stage` argument and make less magic below if stage is not passed.